.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      DOCKER_BUILDTAGS: "include_oss include_gcs"
+      BUILDTAGS: "include_oss,include_gcs"
```

Comment:
We should remove the oss tag: https://github.com/distribution/distribution/pull/3985

Reply:
It's removed on main, but this PR is targeting v2.8 which still includes oss support.